### ISSUE TITLE
Blob is not defined (nextjs - ssr)

### ISSUE THREAD

**Issue by joshuaellis** [pmndrs/drei#233]
- `three` version: 124
- `react-three-fiber` version: 5.3.12
- `@react-three/drei` version: 2.2.16

### Problem description:

with the new update I can see `Cloud.tsx` imports `CloudImage` which at some point calls `new Blob` I don't think `Blob` is defined in `node`.

### Suggested solution:

we could use `cross-blob` which is probably like `isomorphic-fetch`. Alt we can look at other ways that mean we don't need to use another library.

**Comment by joshuaellis** [comment]
used `blob-polyfill` as `fetch-blob` didn't work in browser probs not isomorphic.

**Comment by joshuaellis** [comment]
needs to add polyfill for `atob`: 
```ts
const atobPolyfill = a => Buffer.from(a, 'base64').toString('binary')
```

**Comment by joshuaellis** [comment]
fixed in `2.3.0-beta.3`
